### DISCUSSION TITLE
Remember me checkbox not working

### DISCUSSION THREAD

**Discussion by leeuweriq** [spatie/flareapp.io-roadmap#65] [Ideas]
Hi there,

Ever since we started working with flareapp.io we noticed that the "remember me" checkbox doesn't work. After like 30 minutes or so we need to log in again which is kind of a pain :)  We use Chrome latest as our browser on Mac OS. We have this issue on multiple macs.

**Comment by freekmurze** [comment]
@AlexVanderbist could you take a look?

**Comment by AlexVanderbist** [comment]
Hey @leeuweriq, we've just deployed a fix that should fix this issue! If you're still experiencing problems after today (and maybe a cache purge) feel free to let us know here or on support@flareapp.io. Thanks!

**Comment by leeuweriq** [comment]
Awesome! Thanks for the quick follow-up. Will let you know right away if the issue remains.
